pyproject.toml

```diff
@@ -32,16 +32,16 @@ dependencies = [
   "python-schema-registry-client>=2.4.1",
   "pyyaml>=6.0",
   "rich>=12.4.4",
-  "typer[all]>=0.12.5",
+  "typer>=0.12.5",
```

Comment:
I assume by default, all extras are installed or extra definitions changed in versions in the meantime!?

Reply:
something must've changed meanwhile, all I know is that an extra `all` does not exist anymore 🤷🏻‍♂️

Reply:
https://github.com/fastapi/typer/discussions/785#discussioncomment-9016718